crates/build/re_dev_tools/src/build_web_viewer/lib.rs

```diff
@@ -173,10 +173,7 @@ pub fn build(
         match target {
             Target::Browser => bindgen_cmd.no_modules(true)?.typescript(false),
             Target::Module => bindgen_cmd.no_modules(false)?.typescript(true),
-            Target::NoModulesBase => bindgen_cmd
```

Comment:
Why remove `reference_types`?

Reply:
it's deprecated with the explanation that it's now automatically detected